plugin/plugins/bilibili_danmaku/danmaku_core.py

```
@@ -773,6 +864,8 @@ async def send_danmaku(
     async def stop(self):
         """断开连接（可在任意时刻安全调用，包括连接建立过程中）"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)